### PR TITLE
Remove storage cell blinking code

### PR DESCRIPTION
**Pull request by tth05**
Completely remove old cell blinking code (there is no rendering code for this anymore). This also gets rid of the DriveWatcher class which spammed block updates on almost every insert and extract. The ME Drive/Chest now just update themselves if their state changed every 15 ticks.

### CODE REVIEW

**Comment by Caedis** [comment]
but mah blinkenlights

**Comment by Pelotrio** [comment]
> but mah blinkenlights

There never were any

**Comment by S4mpsa** [comment]
tth coming in with another optimization pass?

**Comment by tth05** [comment]
> tth coming in with another optimization pass?

I knew about this for a while but haven't gotten to it yet. Maybe I'll look at more stuff

**Review by mitchej123 — APPROVED** [review]
seems fine - yolo

File: src/main/java/appeng/api/storage/ICellContainer.java [hunk 1]
```diff
@@ -19,11 +19,4 @@
  * Represents an {@link appeng.api.networking.IGridHost} that contributes to storage, such as a ME Chest, or ME Drive.
  */
 public interface ICellContainer extends IActionHost, ICellProvider, ISaveProvider {
-
-    /**
-     * tell the Cell container that this slot should blink, the slot number is relative to the
-     *
-     * @param slot slot index
-     */
-    void blinkCell(int slot);
```
Comment (mitchej123) [inline src/main/java/appeng/api/storage/ICellContainer.java:28]: hmm, now that I think about this, probably best to make this a default that does nothing as to keep public API compatability

**Review by mitchej123 — CHANGES_REQUESTED** [review]

File: src/main/java/appeng/api/implementations/tiles/IChestOrDrive.java [hunk 2]
```diff
@@ -42,10 +42,4 @@ public interface IChestOrDrive extends ICellContainer, IGridHost, IOrientable {
      * @return if the device is online you should check this before providing any other information.
      */
     boolean isPowered();
-
-    /**
-     * @param slot slot index
-     * @return is the cell currently blinking to show activity.
-     */
-    boolean isCellBlinking(int slot);
```
Comment (mitchej123) [inline src/main/java/appeng/api/implementations/tiles/IChestOrDrive.java:50]: same here; mark it as deprecated but put in a default method for backwards compat

**Review by mitchej123 — APPROVED** [review]

**Pull request merged**